tap_pipedrive/tap.py

```diff
@@ -17,6 +23,76 @@
 logger = singer.get_logger()
 
 
+ERROR_CODE_EXCEPTION_MAPPING = {
+    400: {
+        "raise_exception": PipedriveBadRequestError,
+        "message": "Request is missing or has a bad parameter."
+    },
+    401: {
+        "raise_exception": PipedriveUnauthorizedError,
+        "message": "Invalid authorization credentials."
+    },
+    402: {
+        "raise_exception": PipedrivePaymentRequiredError,
+        "message": "Company account is not open (possible reason: trial expired, payment details not entered)."
+    },
+    403: {
+        "raise_exception": PipedriveForbiddenError,
+        "message": "Invalid authorization credentials or permissions."
+    },
+    404: {
+        "raise_exception": PipedriveNotFoundError,
+        "message": "The requested resource doesn't exist."
```

Comment:
Change doesn't to "does not"

Reply:
Updated error message as per suggestion.